lang/en/http-statuses.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+declare(strict_types=1);
+
+return [
+    '0'            => 'Unknown Error',
```

Comment:
```suggestion
    '0' => 'Unknown Error',
```
fix tương tự các chỗ khác

Reply:
chưa fix đã ressolved là sao?

Reply:
@manhtd-1083 e chỉnh bị thiếu file ở lang/en ạ, e sửa lại ngay ạ